### DISCUSSION TITLE
Add option to don't include SketchwareUtil, SketchLogger, SketchApplication, DebugActivity and FileUtil

### DISCUSSION THREAD

**Discussion by oong819** [Sketchware-Pro/Sketchware-Pro#793] [Ideas]
it's annoying sometimes, if SketchwareUtil excluded, replace SketchwareUtil toast with Android's toast (or just remove it and let user use custom block) and same with other

**Comment by moderGamer** [comment]
> 

using Custom Block

**Comment by j-px** [comment]
Replacing SketchwareUtil.toast() calls won't work all the time since we can only do string-based replacements at the moment, not code-aware replacement like modern IDEs.

The base feature itself, allowing not generating some files, is a good one though.
